sysdump/sysdump.go

```diff
@@ -2098,7 +2098,11 @@ func (c *Collector) SubmitTetragonBugtoolTasks(pods []*corev1.Pod, tetragonAgent
 // (e.g. "a/b/c" => "b/c"). Don't pass an absolute path. It doesn't do what
 // you think it does.
 func removeTopDirectory(path string) (string, error) {
-	index := strings.IndexByte(path, filepath.Separator)
```

Comment:
just curious why this doesn't work. `filepath.Separator` is already defined to be `\\` on windows right?

Reply:
Yeah, for Windows users `filepath.Separator==\` but cilium dump is from Linux host and inside we have `/`.
So, we can hardcode `filepath.Separator` to `/` but what if in the future Cilium will run on Windows Server? )))

Reply:
Yeah so `file path.Separator` is based on the client device's OS. So basically on windows it will be `\` but on Unixes it will be `/`. The bugtool archive will always contain `/` since it runs on Linux, but when we try to remove the separator on a windows host, it will look for `\` instead.